### DISCUSSION TITLE
Json documentation?

### DISCUSSION THREAD

**Discussion by coolcatcoder** [Chlumsky/msdf-atlas-gen#102] [Q&A] [answered]
Is there documentation anywhere, about what the json fields mean?

(Specifically, I would like to know how to get uv coordinates and glyph scale (Scale, so a multiplier of the typical size.) for each glyph.)
(Documentation of all the fields would be nice though. I enjoy reading documentation.)

From reading other discussions, and using the little bit of common sense that I have, I think I partially understand some of it:

- A glyph's plane bounds seems like it would be referring to size via position of edges, but due to some of them having negative values, they do not appear to be a multiplier, and instead are probably within some range, using some units.
- Unicode can be converted to and from characters easily. I understand this one fully.

(Also, for numerical json fields, would the information of what type they are (e.g. 32 bit floats, unsigned 64 bit integers, etc) be available? Not as important, I can work it out with trial and error, but it would be a nice to have.)

**Comment by Chlumsky** [comment, marked as answer]
I have added a brief explanation of the JSON fields to the documentation. All numeric values except indices originate as 64-bit floats. Let me know if anything else is still unclear.
